proposals/csharp-8.0/ranges.md

```diff
@@ -223,7 +223,7 @@ This value will be re-used in the calculation of the second `Slice` argument. Wh
 - When `expr` is of the form `expr1..` (where `expr1` can be omitted), then it will be emitted as `receiver.Length - start`.
 - Otherwise, it will be emitted as `expr.End.GetOffset(receiver.Length) - start`.
 
-The `receiver`, `Length` and `expr` expressions will be spilled as appropriate to ensure any side effects are only executed once. For example:
+The `receiver`, `Length`, and `expr` expressions will be spilled as appropriate to ensure any side effects are only executed once. For example:
```

Comment:
Maybe you should remove the `,` character.

Reply:
That's an Oxford comma; many would suggest it's required there.

Reply:
@theunr

> 
> 
> That's an Oxford comma; many would suggest it's required there.

Thank you